rpcs3/rpcs3qt/qt_utils.cpp

```diff
@@ -183,7 +183,7 @@ namespace gui
 			if (!table)
 				return;
 
-			int item_count = table->rowCount();
+			unsigned int item_count = table->rowCount();
```

Comment:
This doesnt make any sense. It would be better to just use an int in the for loop later

Reply:
Thought that row count would be unsigned, but apparently it isn't